packages/components/data-table/src/data-row.tsx

```diff
@@ -52,7 +92,7 @@ const DataRow = (props) => {
               ? () => props.onRowClick?.(props.row, props.rowIndex, column.key)
               : undefined
           }
-          shouldClipContent={props.shouldClipContent}
+          // shouldClipContent={props.shouldClipContent}
```

Comment:
? 🤔

Reply:
This isn't used in the implementation of the Row component. I guess the author forgot it there. 
I've removed it.